src/libraries/Microsoft.Extensions.DependencyInjection/Microsoft.Extensions.DependencyInjection.sln

```diff
@@ -1,4 +1,8 @@
-﻿Microsoft Visual Studio Solution File, Format Version 12.00
+﻿
```

Comment:
I recommend reverting this file since it's just noise; if left, it will eventually be replaced by an auto-generated one later on.

Reply:
Reverted